mitodl/ocw-studio
# Allow WebsiteContent file upload fields to have any name, not just "file"

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #328 

#### What's this PR do?
Allows the file input field on a content form to have any name, not just "file".  This was more complicated than it first seemed.

A separate PR will be needed to add the file url to the metadata written to the backend (ie git).

#### How should this be manually tested?
Make a copy of the `OCW Hugo Course Starter` but change the name of the Resource file field to "image":
  ```
        {
          "label": "File",
          "name": "image",
          "widget": "file"
        },
  ```        
Create a new Resource with a file.  Check the `WebsiteContent` object in django admin, it should have a populated file field.
Edit the same resource.  You should see the current filename under the file input field and the link should still work.  
Upload a different file and save.  Check in django admin that the file has been updated.
